luts.rs

```diff
@@ -0,0 +1,49 @@
+// This is indeed a copy of Pimoroni's Inky LUTs. Saved here for reference.
+// They license it out under the MIT license. Refer to LICENSE-MIT for more details.
```

Comment:
We should credit Pimoroni in the LICENSE, please add

> Copyright (c) 2018 Pimoroni Ltd.

to the top of LICENSE-MIT